Update deployment-guide-enrollment-mamwe.md (#3407)
MicrosoftDocs/memdocs: memdocs/intune/fundamentals/deployment-guide-enrollment-mamwe.md

> [!TIP]
> [!INCLUDE [tips-guidance-plan-deploy-guides](../includes/tips-guidance-plan-deploy-guides.md)]

> [!IMPORTANT]

> > Mobile Application Management (MAM) is deprecated for Windows. After December 2022 customers will not be able to configure any new WIP policies for devices registered via MEM without enrollment. This does not apply to devices enrolled into MEM.

Reviewer: ```suggestion
> Mobile Application Management (MAM) is deprecated for Windows. After December 2022 customers will not be able to configure any new Microsoft Information Protection (WIP) policies for devices registered via MAM without enrollment. This does not apply to devices enrolled in MAM.
```
